common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -120,6 +124,8 @@
                 collection: this.discussion,
                 course_settings: this.courseSettings,
                 topicId: discussionId,
+                mode: this.mode,
```

Comment:
Does mode still need to be added? I think only start_header is referenced now (and this impacts a few files).